.github/workflows/release.yml

```diff
@@ -98,12 +98,171 @@ jobs:
           cat layer-note
       - name: generate tf layer
         run: |
-          echo "tf layers" >> layer-tf
+          echo "locals {" >> layer.tf
+          echo "  sdk_layer_arns = {" >> layer.tf
           cd ${{ github.event.inputs.layer_name }}
           for file in *
           do
           read arn < $file
-          echo "\""$file"\" = \""$arn"\"" >> ../layer-tf
+          echo "    \""$file"\" = \""$arn"\"" >> ../layer.tf
           done
           cd ..
-          cat layer-tf
+          echo "  }" >> layer.tf
+          echo "}" >> layer.tf
+          cat layer.tf
+      - name: upload layer tf file
+        uses: actions/upload-artifact@v2
+        with:
+          name: layer.tf
+          path: layer.tf
+  smoke-test:
+    runs-on: ubuntu-latest
+    needs: generate-note
+    name: Smoke Test - ${{ matrix.aws_region }} - ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        aws_region: ${{fromJson(github.event.inputs.aws_region).aws_region}}
+        # We need to define the raw parameters to the matrix here but will customize each
+        # below.
+        name:
```

Comment:
actually we only need run smoke test for one specific layer kind, any good idea to skip the others?